### PR TITLE
Add UUID generator

### PR DESCRIPTION
**Pull request by saviomartin**
@tasnimzotder 
**Hi bro**

Added UUID generator in `javascript/uuidgenerator.js`

**Hope it helps
Savio Martin**

### CODE REVIEW

File: javascript/uuidgenerator.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+// npm install uuid
+
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// using version 4 to create uuid
+
+// generates a random uuid
+
+const { v4: uuidv4 } = require('uuid');
```
Comment (tasnimzotder) [inline javascript/uuidgenerator.js:11]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

**Comment by tasnimzotder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261&bid=20442107)

File: javascript/uuidgenerator.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+console.log(`${uuidv4()}`)
```
Comment (tasnimzotder) [inline javascript/uuidgenerator.js:12]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

Comment (tasnimzotder) [inline javascript/uuidgenerator.js:12]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

**Comment by tasnimzotder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261&bid=20442107)

File: javascript/uuidgenerator.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+// npm install uuid
+
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// using version 4 to create uuid
+
+// generates a random uuid
+
+`` const { v4: uuidv4 } = require('uuid'); ``
```
Comment (tasnimzotder) [inline javascript/uuidgenerator.js:11]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

File: javascript/uuidgenerator.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+/* console.log(`${uuidv4()}`) */
```
Comment (tasnimzotder) [inline javascript/uuidgenerator.js:12]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

Comment (tasnimzotder) [inline javascript/uuidgenerator.js:12]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

**Comment by saviomartin** [comment]
I have changed it, still errors, how

**Comment by tasnimzotder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261&bid=20442107)

File: javascript/uuidgenerator.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// here we use only v4,
+// to use v5 uncomment it using ctrl + /
+// using version 5 to create uuid
+// generates a random uuid
+
+const { v5: uuidv5 } = require('uuid');
```
Comment (tasnimzotder) [inline javascript/uuidgenerator.js:24]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

File: javascript/uuidgenerator.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+// npm install uuid
+
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// using version 4 to create uuid
+// generates a random uuid
+
+const { v4: uuidv4 } = require('uuid');
+console.log(`${uuidv4()}`);
```
Comment (tasnimzotder) [inline javascript/uuidgenerator.js:11]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

File: javascript/uuidgenerator.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// here we use only v4,
+// to use v5 uncomment it using ctrl + /
+// using version 5 to create uuid
+// generates a random uuid
+
+const { v5: uuidv5 } = require('uuid');
+console.log(`${uuidv5()}`);
```
Comment (tasnimzotder) [inline javascript/uuidgenerator.js:25]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

File: javascript/uuidgenerator.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
+
+// npm install uuid
+
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// using version 4 to create uuid
+// generates a random uuid
+
+const { v4: uuidv4 } = require('uuid');
```
Comment (tasnimzotder) [inline javascript/uuidgenerator.js:10]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

**Comment by tasnimzotder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261&bid=20442107)

File: javascript/uuidgenerator.js [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// here we use only v4,
+// to use v5 uncomment it using ctrl + /
+// using version 5 to create uuid
+// generates a random uuid
+
+const { v5: uuidv5 } = require("uuid");
+Console.log(`${uuidv5()}`);
```
Comment (tasnimzotder) [inline javascript/uuidgenerator.js:26]: Codacy found an issue: ['Console' is not defined.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

File: javascript/uuidgenerator.js [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+// npm install uuid
+
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// using version 4 to create uuid
+// generates a random uuid
+
+const { v4: uuidv4 } = require("uuid");
+Console.log(`${uuidv4()}`);
```
Comment (tasnimzotder) [inline javascript/uuidgenerator.js:12]: Codacy found an issue: ['Console' is not defined.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

**Comment by tasnimzotder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261&bid=20442107)

File: javascript/uuidgenerator.js [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// here we use only v4,
+// to use v5 uncomment it using ctrl + /
+// using version 5 to create uuid
+// generates a random uuid
+
+const MY_NAMESPACE = '1b671a64-40d5-491e-99b0-da01ff1f3341';
+console.log(uuidv5('Hello, World!', MY_NAMESPACE));
```
Comment (tasnimzotder) [inline javascript/uuidgenerator.js:27]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

File: javascript/uuidgenerator.js [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+// npm install uuid
+
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// using version 4 to create uuid
+// generates a random uuid
+
+const { v4: uuidv4, v5: uuidv5 } = require('uuid');
+console.log(uuidv4());
```
Comment (tasnimzotder) [inline javascript/uuidgenerator.js:13]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)

**Review by tasnimzotder — APPROVED** [review]

**Pull request merged**